lib/config/configuration.go

```diff
@@ -897,14 +905,29 @@ func applyGoogleCloudKMSConfig(kmsConfig *GoogleCloudKMS, cfg *servicecfg.Config
 	return nil
 }
 
+func validatePROXYProtocolValue(p multiplexer.PROXYProtocolMode) error {
```

Comment:
I've put it there because it only really affects input file config. Fundamentally `PROXYProtocolModeUnspecified` is not an invalid value for the `PROXYProtocolMode` type.